### PR TITLE
Update allowlistedWords

### PR DESCRIPTION
**Pull request by HatScripts**
- [x] I agree to license my contribution under GPL-3.0 and agree to allow distribution on app stores as outlined in [LICENSE-APPSTORE](https://github.com/ajayyy/DeArrow/blob/master/LICENSE-APPSTORE.txt)

To test this pull request, follow the [instructions in the wiki](https://github.com/ajayyy/SponsorBlock/wiki/Testing-a-Pull-Request).

***

I added 110 more terms to the `allowlistedWords` Set in [titleFormatterData.ts](https://github.com/ajayyy/DeArrow/blob/master/src/titles/titleFormatterData.ts).

Edit: I didn't change `fancyTextConversions` but for some reason in the commit it shows as being modified.

### CODE REVIEW

**Comment by ajayyy** [comment]
How did you generate this? It contains incorrect stuff like "ABC"

**Comment by HatScripts** [comment]
> How did you generate this? It contains incorrect stuff like "ABC"

"ABC" was already on the list. I only added a comma after it.

**Comment by HatScripts** [comment]
As for how I generated it, I've been compiling a list of YouTube titles that have errors, such as certain acronyms not being capitalized. I also added other common acronyms.

File: src/titles/titleFormatterData.ts [hunk 1]
```diff
@@ -274,4 +388,4 @@ export const notStartOfSentence = new Set([
...
...,"X"],["𝜲","X"],["𐌢","X"],["𝖷","X"],["𝑋","X"],["𝕏","X"],["𐊴","X"],["𝗫","X"],["🆇","X"],["🅧","X"],["Ⓧ","X"],["𝓧","X"],["Ｘ","X"],["𝒳","X"],["ˣ","X"],["⒳","X"],["Ⅹ","X"],["╳","X"],["ⅹ","X"],["ⵝ","X"],["𝙓","X"],["𝚇","X"],["𝐗","X"],["ⅹ","x"],["𝑥","x"],["𝒙","x"],["𝓍","x"],["𝗑","x"],["𝘅","x"],["ⓧ","x"],["ｘ","x"],["×","x"],["ₓ","x"],["𝘹","x"],["𝐱","x"],["𝚡","x"],["𝔁","x"],["𝙭","x"],["𝒴","Y"],["🄨","Y"],["𝓨","Y"],["𝖸","Y"],["𝘠","Y"],["𝙔","Y"],["𝚼","Y"],["𝛶","Y"],["𝝪","Y"],["𝞤","Y"],["𝚈","Y"],["𝑌","Y"],["𝗬","Y"],["𝐘","Y"],["𝒀","Y"],["𝜰","Y"],["𐊲","Y"],["🆈","Y"],["🅨","Y"],["Ⓨ","Y"],["𝐲","y"],["𝑦","y"],["𝒚","y"],["𝓎","y"],["𝔂","y"],["𝗒","y"],["𝘆","y"],["𝘺","y"],["𝙮","y"],["𝚢","y"],["ʏ","y"],["ỿ","y"],["ꭚ","y"],["ℽ","y"],["𝛄","y"],["𝛾","y"],["𝜸","y"],["𝝲","y"],["𝞬","y"],["🅈","y"],["ⓨ","y"],["ｙ","y"],["ƴ","y"],["ʸ","y"],["⒴","y"],["🄩","Z"],["🅉","Z"],["ꓜ","Z"],["𝗭","Z"],["𝐙","Z"],["𝘡","Z"],["🆉","Z"],["🅩","Z"],["Ⓩ","Z"],["Ｚ","Z"],["ᶻ","Z"],["⒵","Z"],["ℤ","Z"],["𝑍","z"],["𝒁","z"],["𝒵","z"],["𝓩","z"],["𝙕","z"],["𝚉","z"],["𝚭","z"],["𝛧","z"],["𝜡","z"],["𝝛","z"],["𝞕","z"],["𝐳","z"],["𝑧","z"],["𝒛","z"],["𝓏","z"],["𝗓","z"],["𝘇","z"],["𝘻","z"],["𝙯","z"],["𝚣","z"],["ⓩ","z"],["ｚ","z"],["ᴢ","z"],["𝔃","z"]]`));
```
Comment (ajayyy) [inline src/titles/titleFormatterData.ts:391]: What was changed here?

Comment (HatScripts) [reply src/titles/titleFormatterData.ts:391]: I didn't change anything there. From my OP:

>Edit: I didn't change fancyTextConversions but for some reason in the commit it shows as being modified.

Comment (sebm253) [reply src/titles/titleFormatterData.ts:391]: newline at the EOF

**Pull request merged**